thorlcr/graph/thgraphmaster.cpp

```diff
@@ -1054,7 +1054,7 @@ class CThorCodeContextMaster : public CThorCodeContextBase
             r->getResultUnicode(MemoryBuffer2IDataVal(result));
             tlen = result.length()/2;
             tgt = (UChar *)malloc(tlen*2);
-            memcpy(tgt, result.toByteArray(), tlen*2);
+            memcpy_iflen(tgt, result.toByteArray(), tlen*2);
```

Comment:
@ghalliday - looks fine, but perhaps could be:
```
tlen = result.length()/2;
if (tlen)
{
    tgt = (UChar *)malloc(result.length());
    memcpy(tgt, result.toByteArray(), result.length());
}
else
    tgt = nullptr;
```

Reply:
could be, but it is more complicated.